src/app/public/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -306,11 +306,20 @@ export class SkyPhoneFieldInputDirective implements OnInit, OnDestroy, AfterView
       const numberObj = this.phoneUtils.parseAndKeepRawInput(phoneNumber,
         this.phoneFieldComponent.selectedCountry.iso2);
       if (this.phoneUtils.isPossibleNumber(numberObj)) {
-        if (this.phoneFieldComponent.selectedCountry.iso2 !== this.phoneFieldComponent.defaultCountry) {
-          return this.phoneUtils.format(numberObj, PhoneNumberFormat.INTERNATIONAL);
-        } else {
-          return this.phoneUtils.format(numberObj, PhoneNumberFormat.NATIONAL);
+        switch (this.phoneFieldComponent.returnFormat) {
+          case 'international':
+            return this.phoneUtils.format(numberObj, PhoneNumberFormat.INTERNATIONAL);
+          case 'national':
+            return this.phoneUtils.format(numberObj, PhoneNumberFormat.NATIONAL);
+          case 'reactive':
```

Comment:
I would lean toward naming this `'default'` for two reasons: 1) it's immediately obvious to the consumer that it's the default formatting behavior, and 2) the word "reactive" has special meaning in Angular so we wouldn't want to create confusion, especially since this is also a form control.

Reply:
I went back and forth but decided against `default` in case we ever decide to change which of the behaviors is the `default` behavior

Reply:
I'm not sure I follow. Any change to the default behavior would be a breaking change. At the very least, we should find a synonym to `'reactive'` so it doesn't confuse with "reactive" Angular forms. Something like `'automatic'`, `'smart'`, etc.

Reply:
I went ahead and updated that to `default` as you are correct about changing that would be a breaking change anyway.